src/z3c/recipe/sphinxdoc/__init__.py

```diff
@@ -168,9 +171,11 @@ def install(self):
     update = install
 
 
-def main(projects):
+def main(projects, argv=None, exit_on_error=False):
```

Comment:
I don't have an issue for this yet, but our fork of this project goes a step further and needs to handle [bad eggs at script creation time](https://github.com/NextThought/z3c.recipe.sphinxdoc/commit/a09782462f00993817d4f4dc1e35826b275d9177). 

I think this recipe is a "best effort" recipe, not a "fail fast" or "bondage-and-discipline-warnings-are-errors" recipe.